### PR TITLE
Add header count to init request response

### PR DESCRIPTION
**Pull request by sectioneight**
@mranney promises this is the last of the protocol changes :)

@uber/soap

### CODE REVIEW

File: tchannel/messages/init_request.py [hunk 1]
```diff
@@ -39,5 +42,6 @@ def parse(self, payload, size):
 
     def serialize(self, out):
         out.extend(write_number(PROTOCOL_VERSION, self.VERSION_SIZE))
+        out.extend(write_number(len(self.headers), self.NH_SIZE))
         for key, value in self.headers.items():
-            out.extend(write_key_value(key, value, key_size=2))
+            out.extend(write_key_value(key, value, key_size=self.HEADER_SIZE))
```
Comment (sectioneight) [inline tchannel/messages/init_request.py]: Incidental cleanup; trying to remove magic numbers

**Comment by blampe** [comment]
yass

**Comment by mranney** [comment]
Sorry about the late-breaking change. Hopefully it won't 'happen again.

**Comment by sectioneight** [comment]
No worries, just giving you a hard time. This took me all of 3 minutes to do :)

**Pull request merged**